src/software/jetson_nano/thunderloop.cpp

```diff
@@ -7,39 +7,43 @@
 #include "software/jetson_nano/primitive_executor.h"
 #include "software/jetson_nano/services/motor.h"
 #include "software/logger/logger.h"
+#include "software/logger/network_logger.h"
 #include "software/util/scoped_timespec_timer/scoped_timespec_timer.h"
 #include "software/world/robot_state.h"
 
+// 50 millisecond timeout on receiving primitives before we emergency stop the robots
+const double PRIMITIVE_MANAGER_TIMEOUT_NS = 50.0 * MILLISECONDS_PER_NANOSECOND;
+
+const std::string CPU_TEMP_FILE_PATH = "/sys/class/thermal/thermal_zone1/temp";
```

Comment:
any reason that these constants aren't static and not in thunderloop.h?

Reply:
they are local to thunderloop.cpp, but since its a class I've moved it to private consts in thunderloop.h